src/ts/show.ts

```diff
@@ -253,4 +253,29 @@ $(document).ready(function(){
       });
     });
   });
+
+  // Sort column in tabular mode
+  $(".sort-triangle").on("click",(function() {
```

Comment:
this probably won't pass the "yarn jslint" command.

Reply:
Yes i just passed it and saw all the errors. Ill fix it! I just ran out of battery though (at the airport), soon!